ee/tabby-schema/src/schema/auth.rs

```diff
@@ -348,6 +370,75 @@ pub struct UpdateOAuthCredentialInput {
     pub client_secret: Option<String>,
 }
 
+#[derive(GraphQLEnum)]
+pub enum LdapEncryptionKind {
+    None,
+    StartTLS,
+    LDAPS,
+}
+
+#[derive(GraphQLInputObject, Validate)]
+pub struct UpdateLdapCredentialInput {
+    #[validate(length(
+        min = 1,
+        code = "host",
+        message = "host should not be empty and should be a valid hostname or IP address"
+    ))]
+    pub host: String,
+    pub port: i32,
+
+    #[validate(length(min = 1, code = "bind_dn", message = "bind_dn cannot be empty"))]
```

Comment:
```suggestion
    #[validate(length(min = 1, code = "bindDn", message = "bind_dn cannot be empty"))]
```

code need to be camel case (to match frontend field name).

Check other validate derivation for examples